src/nuxeo-cloud/day-to-day/deployment.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Deployment
+review:
+    comment: ''
+    date: '06-19-2020'
+    status: ok
+labels:
+    - nuxeo-cloud
+tree_item_index: 300
+toc: true
+---
+
+## Concept
+
+- **Monitored & Controlled**: The Nuxeo Cloud Operations (NCO) team deploys ALL changes and updates to pre-prod and prod environments
+
+- **Fast**: Requests typically completed within 1-2 days
+
+- **Follow Nuxeo Best Practices**: Deployments done via Marketplace Packages so that our automation process is easily used across all projects
+  - NO Snapshots of Date-Based Releases: only production ready code
+  - DO NOT deploy the same Marketplace Package to the same environment multiple times
+
+- **Use Maintenance Windows / Minimize Impact**: Deployments done during standard maintenance windows unless otherwise specified to minimize potential impact to users.
+
+## Deploy My Configuration
+
+The Nuxeo Cloud Operations (NCO) team will deploy all changes and updates to the pre-prod and prod environments. All deployments are done via Marketplace packages.
```

Comment:
change this section to be

All Nuxeo Cloud Deployments require a JIRA Support Ticket (SUPNXP) to be created with a ticket type of “Nuxeo Cloud Deployment” that provides the following information:
- Name and Link to the correct version of the Marketplace Package
- Required 3rd Party Services or SSO services (e.g. FFMPEG, ImageMagick, Okta, etc.)
- Nuxeo Conf Updates corresponding to packages or 3rd party services
- The Date and Time of the deployment including time zone, which can be done outside of established maintenance windows

Info: Nuxeo Cloud requires a Date and Time for all Deployment Requests, this includes identifying the standard maintenance window as the deployment time.